static/js/index.js

```diff
@@ -685,8 +714,17 @@ ep_comments.prototype.setComment = function(commentId, comment){
   var comments = this.comments;
   comment.date = prettyDate(comment.timestamp);
   comment.formattedDate = new Date(comment.timestamp).toISOString();
+
   if (comments[commentId] == null) comments[commentId] = {};
   comments[commentId].data = comment;
+
+};
+
+ep_comments.prototype.setCommentReply = function(commentReply){
+  var commentReplies = this.commentReplies;
+  var replyId = commentReply[0];
+  if(commentReplies[replyId] == null) commentReplies[replyId] = {}
+  commentReplies[replyId] = commentReply[1];
```

Comment:
we're overriding `commentReplies[replyId]` on the next line, so why do we need `commentReplies[replyId] = {}` when `commentReplies[replyId] == null`?